modules/installation-rhv-downloading-ansible-playbooks.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_rhv/installing-rhv-user-infra.adoc
+
+[id="installation-rhv-downloading-ansible-playbooks_{context}"]
+= Downloading the Ansible playbooks
+
+Download the Ansible playbooks for installing {product-title} version {product-version} on {rh-virtualization}.
+
+.Procedure
+
+. On your installation machine, run the following commands:
++
+[source,terminal]
+----
+$ mkdir playbooks
+$ cd playbooks
+$ curl -L -X GET https://api.github.com/repos/openshift/installer/contents/upi/ovirt\?ref\=$release-{product-version} |
```

Comment:
We can't do this! Github is not an RCM source!
If you want to get the denoted playbooks to do a UPI install you need to get them from an RPM or fileserver on the Red Hat Customer Portal.